apps/coscrad-frontend/src/components/resources/terms/term-detail.presenter.tsx

```diff
@@ -5,4 +5,9 @@ import { GenericDetailPresenter } from '../../../utils/generic-components/presen
 // TODO[https://www.pivotaltracker.com/story/show/183681722] expose commands
 export const TermDetailPresenter = (
     dataAndActions: IDetailQueryResult<ITermViewModel>
-): JSX.Element => <GenericDetailPresenter {...dataAndActions} />;
+): JSX.Element => (
+    <div>
+        <h1>Term: {dataAndActions.data.id}</h1>
```

Comment:
This was just for troubleshooting. We probably do want the user to be able to find the ID to report data issues, but otherwise, the UUIDs won't display well.